### ISSUE TITLE
Reset back to zero

### ISSUE THREAD

**Issue by FreshRigi** [ROFLailXGOD/TimerPause#4]
Hey there. I use this feature a lot for runs that can be done segmented.

My issue is that when a run is over, there isn't an easy way to reset everything back to zero. I manually set the timer back to 0:00, but the timer pause causes the splits to automatically start at the last split. I am running against splits, but I just copy them into a comstum comparison when a run is finished, because I know the other times won't work, so that shouldn't be an issue. In previous times this has happened, I just messed around with it until it worked, but I can't figure out how i did it anymore. Is there a way to just have a complete reset added in this component?

**Comment by ROFLailXGOD** [comment]
Hey Rigi!

Are we talking about a full reset here, as in resetting the timer back to 0 and removing all the splits?

**Comment by FreshRigi** [comment]
Yes, start again from the beginning with timer set to zero, after you complete the final split and finished the run when you have started at a point from where you paused before.

**Comment by ROFLailXGOD** [comment]
Alright, I think this should be doable. Now, I am not sure when I try to implement it, hopefully won't be too long. Thanks for the suggestion.

**Comment by ROFLailXGOD** [comment]
@FreshRigi can you, please, test if this is what you are looking for? It now should reset the timer to 0 when you reset the splits after a finished run. It also will remove all split times if all of them are present a.k.a. you just finished a run.
[Vofel.TimerPause.dll.zip](https://github.com/ROFLailXGOD/TimerPause/files/11722562/Vofel.TimerPause.dll.zip)

**Comment by FreshRigi** [comment]
That's exactly what I was looking for. Thanks a lot!
